src/Illuminate/Queue/SerializesAndRestoresModelIdentifiers.php

```diff
@@ -26,11 +26,11 @@ protected function getSerializedPropertyValue($value)
         }
 
         if ($value instanceof QueueableEntity) {
-            return new ModelIdentifier(
+            return (new ModelIdentifier(
                 get_class($value),
                 $value->getQueueableId(),
                 $value->getQueueableConnection()
-            );
+            ))->setChanges($value->getChanges());
```

Comment:
Should `getChanges` method be added to `\Illuminate\Contracts\Queue\QueueableEntity` interface ?